phpzlc/phpzlc/1.0/manifest.json

```diff
@@ -0,0 +1,16 @@
+{
+    "bundles": {
+        "PHPZlc\\PHPZlc\\Bundle\\PHPZlcBundle": ["all"]
+    },
+    "copy-from-package": {
```

Comment:
https://github.com/symfony/recipes#copy-from-recipe-configurator

You can't do this, whenever a newer package is installed this will overwrite the files you copied from your package
to the local vendor directory.

You would have to synchronize this recipe after every installation. 
Maybe you can explain why you need to do this and find another solution. Overwriting vendor files seems like a big code smell.

Reply:
I agree with @sstok here, This looks super strange. Im not sure a recipe should do all this. 

I also have a hard time reading your docs and what the packages should do as they are documented in non-English.

Reply:
My project is for Chinese users, and the users are not extensive. I am willing to meet the community standards. I hope to get a more specific description, which is very important to me.